### PR TITLE
[ASR] [V2A] Adding properties for VMware DR monitoring

### PR DESCRIPTION
**Pull request by vidyadharijami**
## Description

Added properties for Process server in GetAsrFabric and process server name in GetAsrProtectedItem

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/RecoveryServices/RecoveryServices.SiteRecovery/Models/PSFabricDetails.cs [hunk 1]
```diff
@@ -117,6 +117,14 @@ public ASRProcessServer(ProcessServer processServer)
             this.SystemLoadStatus = processServer.SystemLoadStatus;
             this.TotalMemoryInBytes = processServer.TotalMemoryInBytes;
             this.TotalSpaceInBytes = processServer.TotalSpaceInBytes;
+            this.Health = processServer.Health;
```
Comment (isra-fel) [inline src/RecoveryServices/RecoveryServices.SiteRecovery/Models/PSFabricDetails.cs]: and also update existing tests to test these new properties

**Comment by vidyadharijami** [comment]
> Please add an entry to `ChangeLog.md` as the PR template suggests

Updated ChangeLog.md

File: src/RecoveryServices/RecoveryServices.SiteRecovery/Models/PSFabricDetails.cs (hunk 1 above)
Comment (vidyadharijami) [reply src/RecoveryServices/RecoveryServices.SiteRecovery/Models/PSFabricDetails.cs]: The infrastructure used in previous tests is not present anymore. To add the new properties, I need to add the new infra. Will take this up in separate PR.

**Pull request merged**